app/views/contact/_contact_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for @contact, url: contact_index_path, remote: true do |f| %>
+    <div class="col-md-6">
+        <%= f.label :name %></br>
+        <%= f.text_field  :name, required: true, class: "contact-form-text-area" %></br>
+
+        <%= f.label :email %></br>
+        <%= f.text_field :email, required: true, class: "contact-form-text-area" %></br>
+
+      <%= f.label :message %></br>
+      <%= f.text_area :message, rows: 8, cols: 40, required: true, class: "contact-form-text-area",
+                        placeholder: "Send me a message"%></br>
+
+      <div class= "hidden">
+        <%= f.label :nickname %>
+        <%= f.text_field :nickname, :hint => 'Leave this field blank!' %>
+      </div>
+
+      <%= f.submit 'Send Message', class: 'btn btn-primary' %>
+    </div>
+  <% end %>
+  <div class="col-md-6" id="flash-message">
+    <%= render 'flash' %>
+  </div>
```

Comment:
All of this was gotten from a tutorial, though most of it is pretty standard stuff.